Brokerages/QuantConnect.Brokerages.csproj

```diff
@@ -5,7 +5,7 @@
     <RootNamespace>QuantConnect.Brokerages</RootNamespace>
     <AssemblyName>QuantConnect.Brokerages</AssemblyName>
     <TargetFramework>net462</TargetFramework>
-    <LangVersion>6</LangVersion>
+    <LangVersion>7.3</LangVersion>
```

Comment:
To prevent backwards compatibility and cloud conflicts please leave this at v6,

Reply:
I use features as follows

![image](https://user-images.githubusercontent.com/574267/123557183-1f982e80-d798-11eb-89e1-f72b5c206f37.png)

The code will become longer without them. I'm not sure exactly what do you mean by compatibility (you could bump up dependant libraries) and cloud conflict (make no sense). Please, confirm you're 100% tightened to v6 and will roll it back.

Reply:
Correct. Its required - please leave at v6.

Reply:
Hey! Seems the PR needs a rebase, we currently target net5.0 and are using language v9 in all Lean

Reply:
It requires rebasing indeed. Alas, the blocker is https://github.com/QuantConnect/Lean/pull/5692#issuecomment-869214780 . I'll make another attempt and let you know.